components/common/successfulModal/SingleItemMedia.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <BaseMediaItem class="border border-k-shade" :src="src" preview is-detail />
+
+  <div class="mt-5 border-b-k-shade">
+    <p class="is-size-6 capitalize font-bold text-center">
+      {{ header }}
+    </p>
+    <p class="capitalize text-xs text-center mt-2">
+      {{ $t('drops.artBy', [nftName]) }}
+      <a
+        v-safe-href="collectionUrl"
```

Comment:
please use `NuxtLink`, no need for computed property

Reply:
fixed